GameData/RP-0/Parts/RFProcTanks.cfg

```diff
@@ -423,8 +423,40 @@
 	}
 }
 
+@PART[RO-ProcTankSpaceplane]:FOR[RP-0]
+{
+	@MODULE[ProceduralPart]
+	{
+                !TECHLIMIT,* {}
+                %TECHLIMIT
+                {
+                        name = start
+                        diameterMin = 0.01
+                        diameterMax = Infinity
+                        lengthMin = 0.01
+                        lengthMax = Infinity
+                        volumeMin = 0.001
+                        volumeMax = Infinity
+                }
+	}
+        @MODULE[ModuleFuelTanks]
+        {
+                %maxUtilization = 92
+                %utilization = 92
+        }
+	MODULE
+	{
+		name = ModuleToolingPTank
+		toolingType = TankShielded
+		untooledMultiplier = 0.25
+		finalToolingCostMultiplier = 1.375 // more than balloon, less than SM
+	}
+
+}
+
 @PART[proceduralTankRealFuels]:FOR[zzzRP-0]
 {
 	%category = -1
 	%techRequired = ORPHANS
 }
+
```

Comment:
Why is this line added? No need for it.

Reply:
I honestly cannot remember anymore. I guess it's either carried over from whatever current config I copied, or a remnant of the old config that's escaped deletion.

Reply:
It's OK, it was merged already.